### DISCUSSION TITLE
I want to automatically insert commas every three digits for numerical values in the initial values of cellData.

### DISCUSSION THREAD

**Discussion by IchiharaGaku** [dream-num/univer#2295] [Q&A | 常见问题]
Is there a way to automatically insert commas for numerical values in the initial values of cellData? 
Also, when adding rows, I want specific columns to be formatted so that all numbers have commas every three digits.

Example:
12345 → 12,345
12345678 → 12,345,678
123456789 → 123,456,789

**Comment by jikkai** [comment]
Perhaps this plugin can help: https://univer.ai/guides/sheet/configuration/register-plugin#number-format

**Comment by IchiharaGaku** [comment]
@jikkai 
I know, but this documemt doesn't have how to use this plugin.
I want to know code example of React or Next.js...
